src/_includes/css/reset.css

```diff
@@ -23,11 +20,7 @@ pre {
 
 button,
 input,
-optgroup,
 select,
 textarea {
-  font-family: inherit;
-  font-size: 100%;
-  line-height: 1.4;
-  margin: 0;
+  font: inherit;
```

Comment:
Because `optgroup` inherits the font features from `select` and I thought setting `margin: 0` would be unnecessary, because if someone sets margin on those form elements, it sure be overwritten by it, and setting "margin: 0" in the first place makes it worse.
If someone is going to change the margin -> it will be overwritten,
If not -> it will use browsers default.

Reply:
Fair. As long as this is just a basic boilerplate to create Lume themes, not intended to be a proper reset/normalizer, I'm okay. People will edit the CSS code as they want.